com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/LuaScannerTest.java

```diff
@@ -261,13 +269,92 @@ public void testPropsMaterial() throws Exception {
         assertProperty(properties, "prop4", "material", 3);
     }
 
-    public static String trimLines(String string) {
-        String[] lines = string.split("\\r?\\n");
-        StringBuilder result = new StringBuilder();
-        for (String line : lines) {
-            result.append(line.trim()).append("\n");
-        }
-        return result.toString();
```

Comment:
Using of this function is a bit tricky. It's easier to take into account spaces in expected results